backend/README.md

```diff
@@ -0,0 +1,53 @@
+# Backend
+
+An Express.js server that exposes a GraphQL API.
+
+### First-time Setup
+
+* Install the packages needed to run the backend application with<br>
+  ```
+  $ npm install
+  ```
+* Set up environment variables in `nodemon.json` needed to run all the modules
+  * **MongoDB**
```

Comment:
I think we could just mention that a running mongodb is required, but since you started this nice how to, could you also add option with link and mention local setup of mongodb for development.

Reply:
Should be fine now.